Backoffice/Context/ContextManager.php

```diff
@@ -20,8 +19,6 @@ class ContextManager implements CurrentSiteIdInterface
     protected $siteId;
     protected $session;
     protected $tokenStorage;
-    protected $defaultLocale;
```

Comment:
you removed attribute defaultLocale but in the constructor you set it and use it in the method getCurrentLocale